modules/quickstart-public-cloud/pages/clients.adoc

```diff
@@ -8,7 +8,7 @@ Most new features and enhancements are only available for Salt-managed clients,
 For more details, see xref:client-configuration:supported-features.adoc[]
 
 In the {productname} {webui}, navigate to menu:Systems[Bootstrapping], then fill in the ``Host``, ``SSH Port``, ``User`` and ``Password`` fields.
-Make sure you use stable FQDNs for the ``Host`` field, or {productname} will not be able to find your host when your Public Cloud gives you a different short-lived FQDNS.
+Make sure you use stable FQDNs for the ``Host`` field, or {productname} can not find your host when your Public Cloud gives you a different short-lived FQDNS.
```

Comment:
```suggestion
Make sure you use stable FQDNs for the ``Host`` field, or {productname} cannot find your host when your Public Cloud gives you a different short-lived FQDNS.
```

Reply:
Clearly my brain just refuses to learn this :woman_facepalming: